### ISSUE TITLE
Timeout error scanning cloudfront cname

### ISSUE THREAD

**Issue by Giannandrea** [nabla-c0d3/sslyze#445]
**Describe the bug**
An error "ERROR: Connection to server timed out; discarding scan" occurred scanning cloudfront hostname.

**To Reproduce**
1. Install SSLyze using the method you prefer
2. Run the following command `sslyze --regular pro.beefree.io`
4. See error "ERROR: Connection to server timed out; discarding scan"

**Expected behavior**

**Python environment (please complete the following information):**
 - OS: macOS catalina / ubuntu 20.04
 - Python version: 3.7

**Additional context**
It seems that the problem  is related to TLS1_3 and cloudfront, performing the ServerConnectivityTester (server_connectivity.py). 
`openssl s_client -tls1_3 -connect pro.beefree.com:443` is very slow and go in timeout and produce an error.
As workaround i could increase network_timeout in server_setting.py  to 35 or change         
```
for tls_version in [
            TlsVersionEnum.TLS_1_3,
            TlsVersionEnum.TLS_1_2,
            TlsVersionEnum.TLS_1_1,
            TlsVersionEnum.TLS_1_0,
            TlsVersionEnum.SSL_3_0,
        ]:
```
to
```
for tls_version in [
            TlsVersionEnum.TLS_1_2,
            TlsVersionEnum.TLS_1_1,
            TlsVersionEnum.TLS_1_0,
            TlsVersionEnum.SSL_3_0,
            TlsVersionEnum.TLS_1_3,
        ]:
```
In to the file server_connectivity.py
But i can't find a clean fix for this issue.

**Comment by nabla-c0d3** [comment]
It looks like Cloudfront changed their implementation or configuration of TLS 1.3 and it is now kind of buggy. I've added a workaround so the scans should still work now.

**Comment by nabla-c0d3** [comment]
Released in v3.0.7.

**Comment by Giannandrea** [comment]
@nabla-c0d3 
Could you rebuild the docker image?

```
Giannandrea-MACBOOKPRO:sslyze-test giannandrea$ docker run nablac0d3/sslyze --version
Unable to find image 'nablac0d3/sslyze:latest' locally
latest: Pulling from nablac0d3/sslyze
.....
8209f51d9d78: Pull complete
Digest: sha256:ff2c5c626401b1961736a5b2ae6e35a41d213e8b2712102100abf5ee46dcca71
Status: Downloaded newer image for nablac0d3/sslyze:latest
3.0.6
```
Building the image starting from your Dockerfile version 3.0.7 is installed installed inside the container.
